kotlin-android/app/src/main/java/com/mongodb/tasktracker/model/Task.kt

```diff
@@ -6,7 +6,7 @@ import org.bson.types.ObjectId
 
 enum class TaskStatus(val text: String, val code: Int) {
     OPEN("Open", 0),
-    IN_PROGRESS("In Progress", 1),
+    IN_PROGRESS("InProgress", 1),
```

Comment:
sorry that i don't have any context here, why can't we put a space in this? On Android particularly where we display the status text itself it looks kind of sloppy imo.

Reply:
Addressed, unfortunately the "enum" "should" not contain spaces.